### PR TITLE
Better readme

### PR DESCRIPTION
**Pull request by TheAngryByrd**
## Proposed Changes

Updates the READMEs to reflect the Library/Console Application split.

## Types of changes

What types of changes does your code introduce to MiniScaffold?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] Build and tests pass locally
- [x] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [x] I have added necessary documentation (if appropriate)

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: Content/Console/README.md [hunk 1]
```diff
@@ -40,14 +43,42 @@ $ ./build.sh  // on unix
...
+$ ./build.sh  <optional buildtarget>// on unix
 ```
 
+---
+
+### Build Targets
+
+
+- `Clean` - Cleans up artifact and temp directories
+- `DotnetRestore` - Runs [dotnet restore](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-restore?tabs=netcore2x) on the sln file
```
Comment (jdodds) [inline Content/Console/README.md]: would `the sln file` here and below be more clear as `the solution`?

Comment (TheAngryByrd) [reply Content/Console/README.md]: Yeah.  Do you think I should link to official docs on what a `sln` file is as well?

Comment (jdodds) [reply Content/Console/README.md]: it's kinda on the "people probably already know" level, but it can't hurt :)

Comment (TheAngryByrd) [reply Content/Console/README.md]: done

File: Content/Console/README.md [hunk 2]
```diff
@@ -11,27 +11,30 @@ MacOS/Linux | Windows
...
+[![NuGet Badge](https://buildstats.info/nuget/MyLib.1)](https://www.nuget.org/packages/MyLib.1/) | [![NuGet Badge](https://buildstats.info/nuget/MyLib.1?includePreReleases=true)](https://www.nuget.org/packages/MyLib.1/)
 
-- [VSCode Dev Container](https://code.visualstudio.com/docs/remote/containers)
 
 ---
 
-### Building
+### Developing
 
 Make sure the following **requirements** are installed in your system:
```
Comment (kragt) [inline Content/Console/README.md]: I'd use "installed on your system"

File: Content/Console/README.md [hunk 3]
```diff
@@ -11,27 +11,30 @@ MacOS/Linux | Windows
...
 
-#### Environment Variables
+- [VSCode Dev Container](https://code.visualstudio.com/docs/remote/containers)
+
+
+---
+
+### Environment Variables
 
 - `CONFIGURATION` will set the [configuration](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-build?tabs=netcore2x#options) of the dotnet commands.  If not set it will default to Release.
```
Comment (kragt) [inline Content/Console/README.md]: "If not set it will default" -> "If not set, it will default"

File: Content/Library/README.md [hunk 4]
```diff
@@ -44,14 +43,59 @@ $ ./build.sh  // on unix
 - `DISABLE_COVERAGE` Will disable running code coverage metrics.  AltCover can have [severe performance degradation](https://github.com/SteveGilham/altcover/issues/57) so it worth disabling when looking to do a quicker feedback loop.
   - `DISABLE_COVERAGE=1 ./build.sh`
 
-### Watch Tests
 
-The `WatchTests` target will use [dotnet-watch](https://github.com/aspnet/Docs/blob/master/aspnetcore/tutorials/dotnet-watch.md) to watch for changes in your lib or tests and re-run your tests on all `TargetFrameworks`
```
Comment (TravisTheTechie) [inline Content/Library/README.md]: s/will use/uses. It's not conditional, correct?

File: Content/Console/README.md [hunk 5]
```diff
@@ -40,14 +43,42 @@ $ ./build.sh  // on unix
 - `DISABLE_COVERAGE` Will disable running code coverage metrics.  AltCover can have [severe performance degradation](https://github.com/SteveGilham/altcover/issues/57) so it worth disabling when looking to do a quicker feedback loop.
```
Comment (kragt) [inline Content/Console/README.md]: "so it worth" -> "so it is worth"

File: Content/Console/README.md [hunk 6]
```diff
@@ -40,14 +43,42 @@ $ ./build.sh  // on unix
...
+- `CreatePackages` - Runs the packaging task from [dotnet-packaging](https://github.com/qmfrederik/dotnet-packaging). This creates applications for `win-x64`, `osx-x64` and `linux-x64` - [Runtime Identifiers](https://docs.microsoft.com/en-us/dotnet/core/rid-catalog).  
+    - Bundles the `win-x64` application in a .zip file.
+    - Bundles the `osx-x64` application in a .tar.gz file.
+    - Bundles the `linux-x64` application in a .tar.gz file.
+- `GitRelease` - Creates a commit message with the [Release Notes](https://fake.build/apidocs/v5/fake-core-releasenotes.html) and a git tag via the version in the `Release Notes`
+- `GitHubRelease` - Publishes a [GitHub Release](https://help.github.com/en/articles/creating-releases) with the Release Notes and any nuget packages
+- `FormatCode` - Runs [Fantomas](https://github.com/fsprojects/fantomas) on the sln
+- [`Release`](#Releasing) - Task that runs all release type tasks such as `GitRelease` and `GitHubRelease`. Make sure to read [Releasing](#Releasing) to setup your environment correctly for releases.
+
```
Comment (TravisTheTechie) [inline Content/Console/README.md]: Inconsistent between full sentences and fragments in this list.

File: Content/Library/README.md [hunk 7]
```diff
@@ -44,14 +43,59 @@ $ ./build.sh  // on unix
...
+- `DotnetPack` - Runs [dotnet pack](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-pack). This include running [SourceLink](https://github.com/dotnet/sourcelink)
+- `SourcelinkTest` - Runs a Sourcelink Test tool to verify Sourcelinks were properly generated
+- `PublishToNuget` - Publishes the nuget via [paket push](https://fsprojects.github.io/Paket/paket-push.html)
+- `GitRelease` - Creates a commit message with the [Release Notes](https://fake.build/apidocs/v5/fake-core-releasenotes.html) and a git tag via the version in the `Release Notes`
+- `GitHubRelease` - Publishes a [GitHub Release](https://help.github.com/en/articles/creating-releases) with the Release Notes and any nuget packages
+- `FormatCode` - Runs [Fantomas](https://github.com/fsprojects/fantomas) on the sln
+- [`Release`](#Releasing) - Task that runs all release type tasks such as `PublishToNuget` `GitRelease` and `GitHubRelease`. Make sure to read [Releasing](#Releasing) to setup your environment correctly for releases.
```
Comment (TravisTheTechie) [inline Content/Library/README.md]: Again, inconsistent in fragments vs. full sentences.

File: Content/Library/README.md (hunk 4 above)
Comment (TravisTheTechie) [reply Content/Library/README.md]: Should add it matches the voice used elsewhere as well.

Comment (TheAngryByrd) [reply Content/Library/README.md]: Removed this section entirely so shouldn't be a problem

Comment (TravisTheTechie) [reply Content/Library/README.md]: You know, it's red, this should have been obvious. Ignore me.

**Comment by TravisTheTechie** [comment]
Using [write-good](https://github.com/btford/write-good) [or VS Code [plugin Write Good Linter](https://marketplace.visualstudio.com/items?itemName=travisthetechie.write-good-linter)] can help identify new areas of improvement in writing. (Though there's only a few items it's call outs as possible issues.) 
Screen Shot 2019-10-15 at 6 30 16 PM

File: Content/Console/README.md (hunk 2 above)
Comment (TheAngryByrd) [reply Content/Console/README.md]: thanks a lot!

File: Content/Console/README.md (hunk 3 above)
Comment (TheAngryByrd) [reply Content/Console/README.md]: thanks a lot!

File: Content/Console/README.md (hunk 5 above)
Comment (TheAngryByrd) [reply Content/Console/README.md]: thanks a lot!

File: Content/Console/README.md (hunk 6 above)
Comment (TheAngryByrd) [reply Content/Console/README.md]: You would think speaking english my whole life I wouldn't be so terrible at this.

Comment (TravisTheTechie) [reply Content/Console/README.md]: Look, I spent multiple years where one of my primary job duties was editing and I still feel like I'm at best passable at it. In no way am I good ;)

File: Content/Library/README.md [hunk 8]
```diff
@@ -17,41 +17,85 @@ Stable | Prerelease
...
+- `WatchTests` - Runs [dotnet watch](https://docs.microsoft.com/en-us/aspnet/core/tutorials/dotnet-watch?view=aspnetcore-3.0) with the test projects. Useful for rapid feedback loops.
+- `GenerateAssemblyInfo` - Generates [AssemblyInfo](https://docs.microsoft.com/en-us/dotnet/api/microsoft.visualbasic.applicationservices.assemblyinfo?view=netframework-4.8) for libraries 
+- `DotnetPack` - Runs [dotnet pack](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-pack). This include running [SourceLink](https://github.com/dotnet/sourcelink)
+- `SourcelinkTest` - Runs a Sourcelink Test tool to verify Sourcelinks were properly generated
+- `PublishToNuget` - Publishes the nuget via [paket push](https://fsprojects.github.io/Paket/paket-push.html)
+- `GitRelease` - Creates a commit message with the [Release Notes](https://fake.build/apidocs/v5/fake-core-releasenotes.html) and a git tag via the version in the `Release Notes`
+- `GitHubRelease` - Publishes a [GitHub Release](https://help.github.com/en/articles/creating-releases) with the Release Notes and any nuget packages
```
Comment (andrewpucci) [inline Content/Library/README.md]:
```suggestion
- `GitHubRelease` - Publishes a [GitHub Release](https://help.github.com/en/articles/creating-releases) with the Release Notes and any NuGet packages
```

File: Content/Library/README.md [hunk 9]
```diff
@@ -17,41 +17,85 @@ Stable | Prerelease
...
+- `FormatCode` - Runs [Fantomas](https://github.com/fsprojects/fantomas) on the sln
```
Comment (andrewpucci) [inline Content/Library/README.md]: Maybe add a brief description of Fantomas? Like `Runs Fantomas, a code formatter, on the solution`, so they don't need to click through the link to understand what this is from a high level

File: Content/Library/README.md [hunk 10]
```diff
@@ -17,41 +17,85 @@ Stable | Prerelease
...
+- [`Release`](#Releasing) - Task that runs all release type tasks such as `PublishToNuget` `GitRelease` and `GitHubRelease`. Make sure to read [Releasing](#Releasing) to setup your environment correctly for releases.
```
Comment (andrewpucci) [inline Content/Library/README.md]:
```suggestion
- [`Release`](#Releasing) - Runs all release type tasks such as `PublishToNuget`, `GitRelease`, and `GitHubRelease`. Make sure to read [Releasing](#Releasing) to setup your environment correctly for releases.
```

Suggest removing `Task that` for consistency with the previous bullets. Are the 3 tasks listed the only tasks run with this build target? If so, you should write out the `such as`. If not, is there any reason not to include all tasks that would be run here, for clarity?

File: README.md [hunk 11]
```diff
@@ -1,152 +1,89 @@
...
-
-
-Stable | Prerelease
-:---: | :---:
-[![NuGet Badge](https://buildstats.info/nuget/MiniScaffold)](https://www.nuget.org/packages/MiniScaffold/) | [![NuGet Badge](https://buildstats.info/nuget/MiniScaffold?includePreReleases=true)](https://www.nuget.org/packages/MiniScaffold/)
-
-
+F# Template for
+- creating and publishing [libraries](Content/Library/README.md) targeting .NET Full `net461` and Core `netstandard2.1`
+- creating and publishing [applications](Content/Console/README.md) targeting .NET  Core `netcoreapp3.0`
```
Comment (andrewpucci) [inline README.md]:
```suggestion
- creating and publishing [applications](Content/Console/README.md) targeting .NET Core `netcoreapp3.0`
```

File: README.md [hunk 12]
```diff
@@ -1,152 +1,89 @@
...
+Not so mini anymore...
+
+### All project types
+
+- [Standard project structure](https://docs.microsoft.com/en-us/dotnet/core/porting/project-structure) for your dotnet application
+- [Build Automation](https://en.wikipedia.org/wiki/Build_automation) tool via [FAKE](https://fake.build/)
+- [Package management](https://en.wikipedia.org/wiki/Package_manager) tool via [Paket](https://fsprojects.github.io/Paket/)
+- [Unit Testing](https://en.wikipedia.org/wiki/Unit_testing) via [Expecto](https://github.com/haf/expecto)
+- [Code Coverage](https://en.wikipedia.org/wiki/Code_coverage) via [Altcover](https://github.com/SteveGilham/altcover)
+    - Also builds an html report with [ReportGenerator](https://github.com/danielpalme/ReportGenerator)
```
Comment (andrewpucci) [inline README.md]:
```suggestion
    - Also builds an HTML report with [ReportGenerator](https://github.com/danielpalme/ReportGenerator)
```

File: README.md [hunk 13]
```diff
@@ -1,152 +1,89 @@
...
+- [Code formatting](https://en.wikipedia.org/wiki/Programming_style) style via [Fantomas](https://github.com/fsprojects/fantomas)
+- `Release` build step commits latest [Release Notes](https://fake.build/apidocs/v5/fake-core-releasenotes.html) in the body and creates a [git tag](https://git-scm.com/book/en/v2/Git-Basics-Tagging).
+    - If you [reference a Pull Request](https://github.com/TheAngryByrd/FSharp.Control.WebSockets/blob/master/RELEASE_NOTES.md#021---2019-09-12) in the `Release Notes` it will [update that Pull Request](https://github.com/TheAngryByrd/FSharp.Control.WebSockets/pull/3#ref-commit-142baba) with the version it was released in.
+- `Release` build step publishes a [GitHub Release](https://help.github.com/en/articles/creating-releases) via the  [Release Notes](https://fake.build/apidocs/v5/fake-core-releasenotes.html) and adds any artifacts (nuget/zip/targz/etc).
```
Comment (andrewpucci) [inline README.md]:
```suggestion
- `Release` build step publishes a [GitHub Release](https://help.github.com/en/articles/creating-releases) via the  [Release Notes](https://fake.build/apidocs/v5/fake-core-releasenotes.html) and adds any artifacts (`.nuget`/`.zip`/`.tar.gz`, etc.).
```

File: README.md [hunk 14]
```diff
@@ -1,152 +1,89 @@
...
+- [Continuous integration](https://en.wikipedia.org/wiki/Continuous_integration) via [AppVeyor](https://www.appveyor.com/docs/) (Windows) and [TravisCI] (Linux) or [GitHub Actions](https://github.com/features/actions)
+
+
+### For [Libraries](Content/Library/README.md)
+- Builds for both `net461` and `netstandard2.1` - [Target Frameworks](https://docs.microsoft.com/en-us/dotnet/standard/frameworks)
+    - To build for `net461`
+        - On windows: Have at least .NET Framework 4.6.1 installed
```
Comment (andrewpucci) [inline README.md]:
```suggestion
        - On Windows: Have at least .NET Framework 4.6.1 installed
```

File: README.md [hunk 15]
```diff
@@ -1,152 +1,89 @@
...
+        - On osx/linux: Have [mono](https://www.mono-project.com/download/stable/) installed
```
Comment (andrewpucci) [inline README.md]:
```suggestion
        - On macOS/Linux: Have [mono](https://www.mono-project.com/download/stable/) installed
```

File: README.md [hunk 16]
```diff
@@ -1,152 +1,89 @@
...
+    - To build for `netstandard2.1`
+        - Have [.NET core 3.0](https://dotnet.microsoft.com/download) installed
+- [Sourcelink](https://github.com/dotnet/sourcelink) which enables a great source debugging experience for your users, by adding source control metadata to your built assets
```
Comment (andrewpucci) [inline README.md]:
```suggestion
- [Source Link](https://github.com/dotnet/sourcelink) which enables a great source debugging experience for your users by adding source control metadata to your built assets
```

File: README.md [hunk 17]
```diff
@@ -1,152 +1,89 @@
...
-
-### Build!
-
-```
-> .\build.cmd // on windows
-$ ./build.sh  // on unix
-```
 
-The bin of your new lib should look similar to:
+### Install the [dotnet template](https://docs.microsoft.com/en-us/dotnet/core/tools/custom-templates) from nuget:
```
Comment (andrewpucci) [inline README.md]:
```suggestion
### Install the [dotnet template](https://docs.microsoft.com/en-us/dotnet/core/tools/custom-templates) from NuGet:
```

File: README.md [hunk 18]
```diff
@@ -1,152 +1,89 @@
...
 
-* [Add your nuget API key to paket](https://fsprojects.github.io/Paket/paket-config.html#Adding-a-NuGet-API-key)
+---
 
-```
-paket config add-token "https://www.nuget.org" 4003d786-cc37-4004-bfdf-c4f3e8ef9b3a
-```
+## Builds
 
+MacOS/Linux | Windows
```
Comment (andrewpucci) [inline README.md]:
```suggestion
macOS/Linux | Windows
```

File: Content/Console/README.md [hunk 19]
```diff
@@ -11,43 +11,74 @@ MacOS/Linux | Windows
 [![Travis Badge](https://travis-ci.org/MyGithubUsername/MyLib.1.svg?branch=master)](https://travis-ci.org/MyGithubUsername/MyLib.1) | [![Build status](https://ci.appveyor.com/api/projects/status/github/MyGithubUsername/MyLib.1?svg=true)](https://ci.appveyor.com/project/MyGithubUsername/MyLib.1)
 [![Build History](https://buildstats.info/travisci/chart/MyGithubUsername/MyLib.1)](https://travis-ci.org/MyGithubUsername/MyLib.1/builds) | [![Build History](https://buildstats.info/appveyor/chart/MyGithubUsername/MyLib.1)](https://ci.appveyor.com/project/MyGithubUsername/MyLib.1)  
 
----
+## Nuget
```
Comment (andrewpucci) [inline Content/Console/README.md]:
```suggestion
## NuGet 
```

File: Content/Library/README.md [hunk 20]
```diff
@@ -17,41 +17,85 @@ Stable | Prerelease
...
 
-- `CONFIGURATION` will set the [configuration](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-build?tabs=netcore2x#options) of the dotnet commands.  If not set it will default to Release.
-  - `CONFIGURATION=Debug ./build.sh` will result in things like `dotnet build -c Debug`
+---
+
+### Environment Variables
+
+- `CONFIGURATION` will set the [configuration](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-build?tabs=netcore2x#options) of the dotnet commands.  If not set, it will default to Release.
+  - `CONFIGURATION=Debug ./build.sh` will result in `-c` additions to commands such as in `dotnet build -c Debug`
 - `GITHUB_TOKEN` will be used to upload release notes and nuget packages to github.
```
[the remark this answers is not in the document]
Comment (TheAngryByrd) [reply Content/Library/README.md]: I do have it in the Releasing Section "Create a GitHub OAuth Token".  Should I double up with it?

**Pull request merged**